recipes/_generate_password.rb

```diff
@@ -17,6 +17,7 @@
 
 execute 'change-admin-password' do
```

Comment:
+1, the PR to fix this issue has been sitting on the foodcritic project and I was hoping it would get merged soon. However, since we are ensuring backwards compatibility I am cool ignoring the rule on this resource.